Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>Girl Develop It! by SG</title>
+</head>
+<body>
+    <!-- Navegacion -->
+    <nav class="container-fluid navbar navbar-light bg-pink navbar-expand">
+        <div class="container">
+            <a href="https://www.girldevelopit.com/" target="_blank" class="navbar-brand d-none d-sm-block sg-brand" >
+
+                <img src="http://www.girldevelopitfargo.com/images/GDI.jpg" alt="GirlDevelopIt_Logo" class="img-thumbnail" >
+            </a>
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link text-light" href="#About">About</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link text-light" href="#Locations">Locations</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link text-light" href="#SignUp">Register</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link text-light" href="#Schedule">Schedule</a>
+                </li>
+            </ul>     
+        </div>  
+    </nav>
+    <div class="container">
+        <h1 class="text-center">Girl Develop It</h1>
+        <!-- Primera sección: About -->
+        <h2 id="About">About</h2>
```

Comment:
Try to always use kebab-case for classes and id's, which is always lower case

Reply:
Will do!